pkg/constants/constants.go

```diff
@@ -35,4 +35,6 @@ const (
 	// that do not specify any priority class and there is no priority class
 	// marked as default.
 	DefaultPriority = 0
+
+	DefaultPodSetName = "main"
```

Comment:
If a constant is a default for an API, I think it's better for it to be in the apis package.

Reply:
Make sense.

Reply:
Make sense.